playbooks/toolbox-mock-images.yaml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Set variables
+  set_fact:
+    toolbox_dirs:
+      "{{ toolbox_dirs }} + [ '{{ zuul.project.src_dir  }}/fedora-toolbox-{{ item }}' ]"
+    toolbox_images:
+      "{{ toolbox_images }} + [ '{{ registry_url }}/f{{ item }}/fedora-toolbox:{{ item }}' ]"
```

Comment:
Good catch. Forgot to update this.

Reply:
It's actually a bit tricky to use the new locations of the images because the Fedora 29 image is still at the old location. I think I managed to get it to work by open-coding the loop and the variables. See https://github.com/containers/toolbox/pull/701